examples/docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: "3.8"
+
+services:
+  ollama:
+    build: ../..  # NOTE: Dockerfile is in the repo root
```

Comment:
Not sure we want to include the build here, it's quite a large build and most users would get stuck I imagine.
```suggestion
```